src/components/App/AppIFrame.js

```diff
@@ -42,6 +42,10 @@ const SANDBOX = [
 ].join(' ')
 
 class AppIFrame extends React.Component {
+  static defaultProps = {
+    onMessage: () => {},
```

Comment:
Totally unnecessary optimization: you could use `noop` from the utils here https://github.com/aragon/aragon/blob/rewrite/src/utils.js#L1

Reply:
Been meaning to use `noop` around, I'll do that :).